src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/ConstructionTapeHelper.java

```diff
@@ -0,0 +1,296 @@
+package com.minecolonies.coremod.entity.ai.citizen.builder;
+
+import com.minecolonies.coremod.blocks.AbstractBlockHut;
+import com.minecolonies.coremod.blocks.ModBlocks;
+import com.minecolonies.coremod.colony.buildings.AbstractBuilding;
+import com.minecolonies.coremod.colony.workorders.WorkOrderBuild;
+import com.minecolonies.coremod.colony.workorders.WorkOrderBuildDecoration;
+import com.minecolonies.coremod.util.BlockUtils;
+import com.minecolonies.coremod.util.StructureWrapper;
+import net.minecraft.block.BlockHorizontal;
+import net.minecraft.block.properties.PropertyDirection;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Helper class to place and remove constructionTapes from the buildings.
+ */
+public final class ConstructionTapeHelper
+{
+    public static final  PropertyDirection FACING     = BlockHorizontal.FACING;
+
+    /**
+     * Private Constructor to hide implicit one.
+     */
+    private ConstructionTapeHelper()
+    {
+        /**
+         * Intentionally left empty.
+         */
+    }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)